runeq/resources/project.py

```diff
@@ -31,6 +335,7 @@ def __init__(
         created_at: float,
         created_by: str,
         updated_by: str,
+        cohorts: Optional[Cohort] = None,
```

Comment:
Does the Project have one patient cohort or multiple? `cohorts` is plural, but its value is a `Cohort` not a `CohortSet`.

Reply:
Ya, its suppose to be cohortset.